dds/DCPS/Util.h

```diff
@@ -140,6 +140,35 @@ void push_back(Seq& seq, const typename Seq::value_type& val)
   seq[len] = val;
 }
 
+// Constructs a sorted intersect of the given two sorted ranges [a,aEnd) and [b,bEnd).
+// (for pre-c++17 code for the similar effect of std::set_intersection in c++17)
+template <typename InputIteratorA, typename InputIteratorB, typename OutputIterator>
+OutputIterator intersect_sorted_ranges(InputIteratorA a, InputIteratorA aEnd,
+                                       InputIteratorB b, InputIteratorB bEnd,
+                                       OutputIterator intersect)
+{
+  while (a != aEnd && b != bEnd) {
+    if      (*a < *b) { ++a; }
```

Comment:
guidelines:
> 
> - ... Do not add extra spaces to make syntax
>   elements (that span lines/statements) line up. This only causes unnecessary
>   changes in adjacent lines as the code evolves.

Reply:
Please update this code per guidelines (see 12/19 comment), or comment here as to why not.